clients/src/main/java/org/apache/kafka/clients/consumer/internals/SubscriptionState.java

```diff
@@ -47,7 +47,7 @@
 import java.util.function.Predicate;
 import java.util.regex.Pattern;
 
-import static org.apache.kafka.clients.consumer.internals.Fetcher.hasUsableOffsetForLeaderEpochVersion;
+import static org.apache.kafka.clients.consumer.internals.DefaultFetcher.hasUsableOffsetForLeaderEpochVersion;
```

Comment:
Can we move this to a different class, perhaps a static class like FetcherUtil?

Reply:
It will be in a follow-up PR.